arquillian-chameleon-deployments/arquillian-chameleon-gradle-build-deployment/src/main/java/org/arquillian/container/chameleon/deployment/gradle/GradleBuild.java

```diff
@@ -0,0 +1,25 @@
+package org.arquillian.container.chameleon.deployment.gradle;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface GradleBuild {
+
+    /**
+     * Gradle Project directory to run the build from
+     * @return radle Project directory relative to current module. By default current directory is used.
+     */
+    String path() default ".";
+
+    /**
+     * In case of disabling quiet mode, Gradle output is printed to console
+     * @return 
+     */
+    // TODO
```

Comment:
Remove this comment?

Reply:
Removed the quiet-parameter altogether for now. Not implemented yet.